.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Your checklist for this pull request
```

Comment:
Hey Matthias thanks for the initiative but I don't think we want such a template right now.

Reply:
thanks for the feedback!